tasks/cockpit-tasks-centosci.yaml

```diff
@@ -18,6 +18,8 @@ spec:
         env:
         - name: TEST_JOBS
           value: '5'
+        - name: TEST_BATCHES
+          value: '2'
```

Comment:
See discussion in the cockpit PR -- I'd rather provide a sane default which ends up as "2" with TEST_JOBS==4 or 5.

Reply:
Agreed, dropped the second commit.